gp_emulator/GaussianProcess.py

```diff
@@ -221,35 +223,113 @@ def predict ( self, testing, do_unc=True ):
 	do_unc: flag, optional
 		Calculate the uncertainty (if you don't set this flag, it
 		can shave a few us"""
-
+       
 
         ( nn, D ) = testing.shape
         assert D == self.D
-        
         expX = np.exp ( self.theta )
         
         a = dist.cdist ( np.sqrt(expX[:(self.D)])*self.inputs, \
             np.sqrt(expX[:(self.D)])*testing, 'sqeuclidean')
-        
         a = expX[self.D]*np.exp(-0.5*a)
         b = expX[self.D]
         
         mu = np.dot( a.T, self.invQt)
+
         if do_unc:
-	    var = b - np.sum (  a * np.dot(self.invQ,a), axis=0)
+	       var = b - np.sum (  a * np.dot(self.invQ,a), axis=0)
```

Comment:
Use a tool such as https://pypi.python.org/pypi/pep8 to check python code meets coding standards.
Once it does, add a pep8 validator check to the Jenkins build.